tests/drivers/counter/counter_basic_api/boards/nrf52811_pca10056.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_COUNTER_TIMER0=y
+CONFIG_COUNTER_TIMER0_PRESCALER=4
+CONFIG_COUNTER_TIMER1=y
+CONFIG_COUNTER_TIMER1_PRESCALER=4
+CONFIG_COUNTER_TIMER2=y
+CONFIG_COUNTER_TIMER2_PRESCALER=4
+CONFIG_COUNTER_RTC0=y
```

Comment:
@nordic-krch @jarz-nordic @ioannisg @anangl I removed `CONFIG_COUNTER_RTC2=y` from here since there's only 2 RTCs in the 8211

Reply:
@carlescufi : Did you remove RTC2 or RTC1 ? Maybe I accidently set RTC2 instead of RTC1. Currently I do not see here RTC1 despite the fact nrf52811 has it.

Reply:
RTC1 is used by the kernel.